sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -197,11 +198,10 @@ def get_all_regions_instance_types_df():
 
 
 # Fetch Images
-REGIONS = ALL_REGIONS
-GPU_TO_IMAGE_DATE = {
+_GPU_TO_IMAGE_DATE = {
     # https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;v=3;search=:64,:Ubuntu%2020,:Deep%20Learning%20AMI%20GPU%20PyTorch # pylint: disable=line-too-long
-    # Commented below are newer AMIs, but as other clouds do not support
-    # torch==1.13.0+cu117 we do not use these AMIs to avoid frequent updates:
+    # Commented below is the new AMI name, but as other clouds do not support
+    # torch==1.13.0+cu117 we do not use this AMI to avoid frequent updates:
```

Comment:
L208-209 are a bit confusing now that L210 specify two timestamps. Not sure how to better phrase it however.